src/main/java/pl/agh/edu/master_diet/core/model/database/Activity.java

```diff
@@ -11,23 +13,15 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class Activity {
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
+public class Activity extends Browsable {
 
     @Column(nullable = false)
-    private String name;
-
-    private String description;
+    private String code;
 
     @Column(nullable = false)
-    private Double burnedCalories;
+    private Double mets;
```

Comment:
Isn't the amount of burned calories also an important information? We might use it in addition to mets, not by replacing burned calories with mets

Reply:
We don't have such information in the dataset. Burned calories depend on weight. Metabolic Equivalent of Task (MET) values are a way to estimate how many calories are burned during a specific physical activity, according to the American Council on Exercise.